microsoftgraph/microsoft-graph-explorer-v4
# Set sample URL regardless of Graph version

## Overview
Fixes #1957 

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions
Provision a test tenant and configure it for canary access or log in with a tenant with canary access
Run any queries
Observe fix
